utils/model_uploader/model_listing/excluded_models.txt

```diff
@@ -0,0 +1,18 @@
+ml-models/huggingface/sentence-transformers/all-MiniLM-L12-v2/1.0.0/onnx
```

Comment:
Can we keep these values in a python file with a constant value? May be we can reuse the same constant for the testing also. And in the python file I would like to have a detailed comment about this exclusion.